src/commands/info.js

```diff
@@ -13,19 +13,51 @@ governing permissions and limitations under the License.
 const {Command, flags} = require('@oclif/command')
 const envinfo = require('envinfo')
 
+const NEW_LINE = '\n'
+const INDENT = (n) => '  '.repeat(n)
+const AIO_PLUGINS = 'AioPlugins'
```

Comment:
Could we rename `AioPlugins` into `CLI Plugins`?
In case we rename the CLI one day ;)